### DISCUSSION TITLE
是否支持流式接口

### DISCUSSION THREAD

**Discussion by fengwei9** [elsa-workflows/elsa-core#5846] [Q&A]
如果支持怎么在设计器中支持，如果不支持后续有机会支持吗

**Comment by sfmskywalker** [comment]
What do you mean with "streaming interface"? Can you elaborate what it is and why you think this is useful?

**Reply by fengwei9** [reply]
因为工作流准备用在chatgpt的问答上，所以看看能不能支持？
